components/faq/FAQ.tsx

```diff
@@ -0,0 +1,16 @@
+import Accordion from '@/components/Accordion/Accordion';
+
+import { faqList } from './FAQList';
+
+/**
+ * This is the FAQ component. It tells the user about the frequently asked questions.
```

Comment:
Specify that this component has been created to render FAQ list inside MDX file.

Reply:
Done